docs/guide/configuring-models.md

```diff
@@ -108,3 +102,17 @@ Additionally, it is possible to write the following:
 ## Database-specific configuration
 
 In addition to the configuration parameters that apply to all database adapters, there are certain configuration options that apply only to specific databases. See the page on [database-specific optimizations](database-optimizations/).
+
+## Hooks
+
+dbt provides the ability to run arbitrary commands against the database before and after a model is run. These are known as pre- and post-model hooks and configured as such:
+
+```YAML
+models:
+  project-name:
+    pre-hook:       # arbitrary SQL
```

Comment:
I think "arbitrary" kind of has a negative connotation. Maybe "custom" instead? "arbitrary" makes me think "arbitrary code execution" which is generally a bad thing. Just my 2 cents